src/ruvsarpur.py

```diff
@@ -480,6 +480,8 @@ def parseArguments():
 
   parser.add_argument("--originaltitle", help="Includes the original title of the show in the filename if it was found (this is usually the foreign title of the series or movie)", action="store_true")
 
+  parser.add_argument("--subtitles", help="Download subtitles if available, for all available languages", action="store_true")
```

Comment:
There is a second utility in this repo to convert webvtt to SRT subtitles. Perhaps doing this conversion could be an option as well?